src/components/PoolingWidget/index.tsx

```diff
@@ -272,6 +291,16 @@ const PoolingInterface: React.FC = () => {
               <SubComponents step={step} {...restProps} />
             </ContentWrapper>
 
+            <LiquidityMessage>
+              <p>
+                Your liquidity is equal to the amount you have deposited into your exchange wallet.
+                <br />{' '}
```

Comment:
Why `{' '}`? It gets collapsed and you don't see it anyway

Reply:
Think it was added by the yarn lint fix. Can remove.